### PR TITLE
Add back 404 page video

### LINKED ISSUE #241: 404 Page is Sad
The old 404 page had a really [great video](https://www.youtube.com/watch?v=t3otBjVZzT0) that would play when the page loaded. The new 404 page is just sad (see the frowny face?)
notfound

I would really appreciate if the 404 page were to be returned to it's former glory. Here's the [old 404 page](https://github.com/ComputerScienceHouse/CSHPublicSite/blob/7324415842296eeca43f9650262e92a091315a62/404.html#L26) source for reference.

### PR DESCRIPTION
**Pull request by rlee80**
Closes #241 

404 Page Web
404 Page Mobille

### CODE REVIEW

**Comment by mxmeinhold** [comment]
Could you enter responsive mode and show how this looks on mobile?

**Comment by mxmeinhold** [comment]
It also looks like you might have changed the indentation?

**Comment by rlee80** [comment]
> Could you enter responsive mode and show how this looks on mobile?

Added

> It also looks like you might have changed the indentation?

If you view the diffs with the view file mode, the indentation is properly formatted

File: 404.html [hunk 1]
```diff
@@ -3,12 +3,20 @@
...
+			</div>
+			<div class="col-xs-12 col-sm-6 col-md-6 vcenter">
+				<div class="embed-responsive embed-responsive-16by9 csh-404-video">
+					<iframe class="embed-responsive-item" src="https://www.youtube.com/embed/t3otBjVZzT0?autoplay=1" frameborder="0" allowfullscreen></iframe>
+				</div>
+			</div>
+			<!---->
+			<div class="col-xs-12 col-sm-6 col-md-6 vcenter">
+
+			</div>
```
Comment (stevenmirabito) [inline 404.html:20]: Why is there an empty `<div>` and an empty comment here?

File: 404.html [hunk 2]
```diff
@@ -3,12 +3,20 @@
...
-</div>
+	<div class="container spaced">
+		<div class="row">
+			<div class="col">
+				<h2>Page not found :(</h2>
+				<p>The requested page could not be found.</p>
+			</div>
+			<div class="col-xs-12 col-sm-6 col-md-6 vcenter">
+				<div class="embed-responsive embed-responsive-16by9 csh-404-video">
+					<iframe class="embed-responsive-item" src="https://www.youtube.com/embed/t3otBjVZzT0?autoplay=1" frameborder="0" allowfullscreen></iframe>
```
Comment (stevenmirabito) [inline 404.html:14]: These `embed-responsive` and `csh-404-video` classes don't seem to be associated with or do anything. Are these necessary?

File: 404.html (hunk 1 above)
Comment (rlee80) [reply 404.html:14]: Fixed

File: 404.html (hunk 2 above)
Comment (rlee80) [reply 404.html]: Fixed, thanks!

File: 404.html [hunk 3]
```diff
@@ -3,12 +3,15 @@
...
+			<div class="col">
+				<h2>Page not found :(</h2>
+				<p>The requested page could not be found.</p>
+			</div>
+			<div>
+				<iframe class="embed-responsive-item" src="https://www.youtube.com/embed/t3otBjVZzT0?autoplay=1" frameborder="0" allowfullscreen></iframe>
+			</div>
+		</div>
+	</div>
 </div>
```
Comment (mxmeinhold) [inline 404.html:17]: It looks like you switched from spaces to tabs. Was that intentional?

Comment (rlee80) [reply 404.html:17]: Yeah

Comment (mxmeinhold) [reply 404.html:17]: It looks like we don't use the tab charachter anywhere else in the repo, would you mind converting back to spaces to keep it consistent?
```
8 20:24 bionicpi@erebus:CSHPublicSite $ grep -rP '\t' .  
9 20:25 bionicpi@erebus:CSHPublicSite $ 
```

Comment (rlee80) [reply 404.html]: > It looks like we don't use the tab character anywhere else in the repo, would you mind converting back to spaces to keep it consistent?
> 
> ```
> 8 20:24 bionicpi@erebus:CSHPublicSite $ grep -rP '\t' .  
> 9 20:25 bionicpi@erebus:CSHPublicSite $ 
> ```

Done!

**Comment by mxmeinhold** [comment]
Looks good to me. Would you squash this down to 1 commit please?

**Pull request merged**